test/controllers/ConfirmWebsiteControllerSpec.scala

```diff
@@ -103,19 +103,19 @@ class ConfirmWebsiteControllerSpec extends ControllerBaseSpec  {
   "Calling the updateWebsite() action in ConfirmWebsiteController" when {
 
     "there is a website in session" when {
-        //TODO uncomment this test when confirmation has been added
-//      "the website has been updated successfully" should {
-//
-//        "show the website changed success page" in {
-//          mockIndividualAuthorised()
-//          mockUpdateWebsite(vrn, testWebsite)(Future(Right(UpdatePPOBSuccess("success"))))
-//          val result = TestConfirmWebsiteController.updateWebsite()(requestWithWebsite)
-//
-//          status(result) shouldBe Status.SEE_OTHER
-//          redirectLocation(result) shouldBe Some("/vat-through-software/account/correspondence/" +
-//            "new-website-address-confirmation")
-//        }
-//      }
+      "the website has been updated successfully" should {
+
+        "show the website changed success page" in {
+          mockIndividualAuthorised()
+
+          mockUpdateWebsite(vrn, testWebsite)(Future(Right(UpdatePPOBSuccess("success"))))
+          val result = TestConfirmWebsiteController.updateWebsite()(requestWithWebsite)
+
+          status(result) shouldBe Status.SEE_OTHER
+          redirectLocation(result) shouldBe Some("/vat-through-software/account/correspondence/" +
```

Comment:
This is tiny but could you change this to be a reverse controller route as opposed to a literal String? Just to make it a bit more safe/clear on where this is being redirected to.